APMrover2/sensors.cpp

```diff
@@ -56,7 +56,7 @@ void Rover::read_sonars(void)
 {
     sonar.update();
 
-    if (sonar.status() == RangeFinder::RangeFinder_NotConnected) {
+    if (sonar.status(0) == RangeFinder::RangeFinder_NotConnected) {
```

Comment:
Rover's not yet making use of the orientation parameter.  Instead it's assuming that if there are two rangefinders, the first one points foward-left, the 2nd one points forward-right.  We should change this so that it uses the orientation but I haven't gotten to that yet.